dotnetv3/KMS/ListKeysExample/ListKeys.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// List the Amazon Key Managements Service keys for the AWS Regsion of the
+/// default user. To list keys in another AWS Region, suplly the region as a
+/// parameter to the client constructor. This example was created using the
+/// AWS SDK for .NET version 3.7 and .NET Core 5.0.
+/// </summary>
+namespace ListKeysExample
+{
+    // snippet-start:[KMS.dotnetv3.ListKeysExample]
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.KeyManagementService;
+    using Amazon.KeyManagementService.Model;
+
+    public class ListKeys
+    {
+        public static async Task Main()
+        {
+            var client = new AmazonKeyManagementServiceClient();
+            var request = new ListKeysRequest();
+            var response = new ListKeysResponse();
+
+            do
+            {
+                response = await client.ListKeysAsync(request);
+
+                response.Keys.ForEach(key =>
+                {
+                    Console.WriteLine($"ID: {key.KeyId}, {key.KeyArn}");
+                });
+
+                // Set the Marker property when response.Truncated is true
+                // in order to get the next keys.
+                request.Marker = response.NextMarker;
+            }
+            while (response.Truncated);
+        }
+    }
+
```

Comment:
Remove this blank line.

Reply:
As I said before that blank line is there because of the linter.